etc/docker-scripts/bootstrapper.py

```diff
@@ -123,36 +123,20 @@ def process_outputs(self) -> None:
         duration = time.time() - t0
         OpUtil.log_operation_info('outputs processed', duration)
 
-    def process_metrics(self) -> None:
-        """Expose metrics and    metadata in the KFP UI
-
-        When invoked this method produces a
-        https://www.kubeflow.org/docs/pipelines/sdk/output-viewer
-        compatible metadata file, which esposes the following information
-        for each pipeline node in the KFP UI:
-         - the name of the input file (e.g. notebook.ipynb)
-         - the name of the input archive (e.g. notebook...tar.gz)
-         - the location of the input archive on COS
-
-        If the notebook | script produced a metrics file it is also
-        exposed in the KFP UI:
-        https://www.kubeflow.org/docs/pipelines/sdk/pipelines-metrics/
-
-        There should be no need to override this method in subclasses
-        because the code is not file type specific.
-        """
+    def process_metrics_and_metadata(self) -> None:
 
         OpUtil.log_operation_info('processing metrics and metadata')
         t0 = time.time()
 
         # Location where the KFP specific output files will be stored
+        # in the environment where the bootsrapper is running
         output_path = Path('/tmp')
```

Comment:
Should we tie this value (`/tmp`) to an env that defaults to `/tmp` in case we run into installations where this is problematic?  I suppose this failure only means they don't get their metrics, but if some metrics are really important, that _could_ override the value via a node-based env entry - provided they know a writable path.

Reply:
good idea! Do we have a naming convention? Maybe name it `ELYRA_WRITABLE_CONTAINER_DIR`? This change does require a corresponding change to https://github.com/elyra-ai/elyra/pull/1054